### ISSUE TITLE
Cannot enter text in the tiny editor's modal box of the moodle site.

### ISSUE THREAD

**Issue by Charlie99009** [arnog/mathlive#2557]
## Description

> When I put a <math-field> into the modal box of the tiny editor on the moodle website, I clicked the middle of the input box, and it showed the cursor but I couldn't enter any text. But when I clicked the edge of the input box or the keyboard button or the menu button, it actually allowed me to enter. After it allowed me to enter, I let it lose focus again, and then clicked the middle of the input box again to let it get focus, and it actually worked.
![1733120743320](https://github.com/user-attachments/assets/34d828d6-cb68-4ef3-8a9f-a6af9ec70370)

### Steps to Reproduce

1. Load the MathLive library from a CDN with a <script> tag on Moodle site.
`<script defer src="//unpkg.com/mathlive"></script>`
2. Add a <math-field> tag in the template file of  a plugin in the tiny editor on Moodle site
`{{< core/modal }}
    {{$body}}
    <math-field class="w-100"></math-field>
    {{/body}}
{{/ core/modal }}`
3. Clear cache and open it .

### Actual Behavior

> When I put a into the modal box of the tiny editor on the moodle website, I clicked the middle of the input box, and it showed the cursor but I couldn't enter any text

### Expected Behavior

> How can I solve the problem of being able to enter text by clicking the middle of the input box when it is first loaded?

### Environment

**MathLive version**  0.102.0.

**Operating System** _macOS.

**Browser** _Safari, Chrome, Edge, Firefox

**Comment by arnog** [comment]
I don't have access to Moodle site. Is this a problem that can be reproduced with Moodle site? If not, could someone familiar with Moodle site and with access to it investigate this issue and isolate if this is a problem with MathLive or somewhere else?

**Comment by Charlie99009** [comment]
> I don't have access to Moodle site. Is this a problem that can be reproduced with Moodle site? If not, could someone familiar with Moodle site and with access to it investigate this issue and isolate if this is a problem with MathLive or somewhere else?

Yes , of course , I think the problem happend in mathlive and the modal box of Moodle, but it has not display any errors on the console .when I put mathlive on page of Moodle(not in modal box) ,its work.

**Comment by arnog** [comment]
Without a reproducible case, I am not able to debug this further.
